### ISSUE TITLE
Custom prefix fails

### ISSUE THREAD

**Issue by iwarshak** [bratsche/twix#6]
Hi - It seems like Twix does not work correctly when custom prefixes are involved. I am running on master, and this is what I get:

```
iex(15)> Twix.tw(["mt-2 block w-full rounded-lg border-zinc-300", "mt-0"])                  
"block w-full rounded-lg border-zinc-300 mt-0"
iex(16)> Twix.tw(["tw-mt-2 tw-block tw-w-full tw-rounded-lg tw-border-zinc-300", "tw-mt-0"])
"tw-mt-2 tw-block tw-w-full tw-rounded-lg tw-border-zinc-300 tw-mt-0"
iex(17)> 
```

Would this be a big effort to support prefixes? I am migrating my app from Bootstrap to Tailwind, so a prefix is necessary in this case.

**Comment by bratsche** [comment]
Yeah, I'm planning on making things more easily configurable. I'll see if I can find some time to work on this soon.

**Comment by iwarshak** [comment]
That'd be much appreciated! I forked the code and started working on it, but it looks more involved than I first thought, and would probably not fit with your vision

**Comment by iwarshak** [comment]
@bratsche 👋 Just wondering if you have had a chance to take a look at this yet?
